app/views/shared/_carousel_item.html.erb

```diff
@@ -106,10 +106,10 @@
           (generic_file.threeD? || !document.read_master?) %> <%# Change conditions when 3D surrogate is available%>
       <%= t('dri.views.catalog.links.download_not_available', index: generic_file_counter.ordinalize) %>
     <% elsif document.read_master? || @surrogate && can?(:read, document.id) || can?(:edit, document.id) %>
-      <%# if generic_file.threeD? %>
-        <%#= link_to t('dri.views.catalog.links.embed_data'), { controller: 'api/oembed', action: 'show', url: catalog_url }, class: 'dri_embed_link' %> <%# Embed link not working for 3D assets %>
-      <%# end %>
-      <%= link_to '#dri_download_modal_id', :'data-bs-toggle' => 'modal', :'data-fileid' => generic_file.id, :id => "configure_download_#{generic_file.id}", class: 'configure_download' do %>
```

Comment:
Is there a reason for removing the generic_file.id from the link id?

Reply:
Not sure if it's being used, but included back anyways. Thanks.